app/models/question.rb

```diff
@@ -21,6 +21,10 @@ class Question < ApplicationRecord
   has_many :taggings, dependent: :destroy
   has_many :tags, through: :taggings
 
+  has_attached_file :audio_prompt
+  validates_attachment_content_type :audio_prompt, content_type: [%r{\Aaudio/.*\Z}, "video/ogg", "video/webm"]
+  validates_attachment_file_name :audio_prompt, matches: /\.(mp3|ogg|webm|wav)\Z/i
```

Comment:
I just realized we need to check what formats Android supports, and they don't seem to support `webm`, so let's remove that one from here and from the hint please.